<!doctype html>

<html>

<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width,initial-scale=1.0">
    <meta name="format-detection" content="telephone=no">

    <title>Saito Lab</title>

    <link rel="stylesheet" href="./slick/slick.css">
    <link rel="stylesheet" href="./slick/slick-theme.css">
    <link rel="stylesheet" href="./css/pc.css" media="screen and (min-width:1201px)">
    <link rel="stylesheet" href="./css/tablet.css" media="screen and (max-width:1200px)and (min-width:601px)">
    <link rel="stylesheet" href="./css/smartphone.css" media="screen and (max-width:600px)">
    <script src="https://code.jquery.com/jquery-3.1.1.js" integrity="sha256-16cdPddA6VdVInumRGo6IbivbERE8p7CQR3HzTBuELA=" crossorigin="anonymous"></script>
    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
    <link href="https://fonts.googleapis.com/css?family=Noto+Sans+JP:400,700&display=swap&subset=japanese" rel="stylesheet">
    <link href="https://fonts.googleapis.com/css?family=Roboto:400,400i,700,700i&display=swap" rel="stylesheet">

    <style>
        li {
          padding: 4px 0;
        }
      </style>
</head>

<body>

    <div id="page">

        <div id="wrapper">

            <header>

               <div id="toparea">

                    <div id="topareatop">

                        <div id="title">
                            <a href="./index-en.html" title="Saito Lab, Kitasato University, AIST & The University of Tokyo" target="_blank"><img src="./images/logo1.png" alt="Saito Lab, Kitasato University, AIST & The University of Tokyo"></a>
                            <h1 class="pt"><a href="./index-en.html" title="Saito Lab, Kitasato University, AIST & The University of Tokyo"><img src="./images/logo2.png" srcset="./images/logo2.png 1x, ./images/logo2@2x.png 2x, ./images/logo2@3x.png3x" alt="Saito Lab, Kitasato University, AIST & The University of Tokyo"></a></h1>
                            <h1 class="sm"><a href="./index-en.html" title="Saito Lab, Kitasato University, AIST & The University of Tokyo"><img src="./images/logo3.png" srcset="./images/logo3.png 1x, ./images/logo3@2x.png 2x, ./images/logo3@3x.png3x" alt="Saito Lab, Kitasato University, AIST & The University of Tokyo"></a></h1>
                        </div>

                        <div id="language"><a href="./publication.html"><img src="./images/japanese.png" srcset="./images/japanese.png 1x, ./images/japanese@2x.png 2x, ./images/japanese@3x.png3x" alt="japanese"></a></div>

                        <div id="humberger" class="hidden-md hidden-lg">
                            <span id="icon-bar1"></span>
                            <span id="icon-bar2"></span>
                            <span id="icon-bar3"></span>
                        </div>

                    </div>

                </div>

                <div id="menuarea">

                    <div id="drawernav" class="hidden-md hidden-lg">
                        <nav id="mainnavigationcontainer" class="box-under-shadow">
                            <ul id="mainnavigation">
                                <li><a href="./index-en.html">HOME</a></li>
                                <li><a href="./research-en.html">Research</a></li>
                                <li><a href="./member-en.html">Member</a></li>
                                <li class="current"><a href="./publication-en.html">Publication</a></li>
                                <li><a href="./joinus-en.html">Join us</a></li>
                                <li><a href="./access-en.html">Contact/Access</a></li>
                                <li><a href="./software-en.html">Software</a></li>
                            </ul>
                        </nav>
                    </div>

                </div>

            </header>

            <div id="main">
                <article>

                    <h2>Students' activities</h2>
                    <p>Our lab encourages students' research publications.<br>
                        Students' papers, conference presentations, and prizes are listed in <a target="_blank" href="./student-en.html">this page</a>.</p>

                    <h2>Papers (peer-reviewed)</h2>
                    <h3>2025</h3>
                    <ul>
                        <li>
                            Jinzen Ikebe, Kazunori Yoshida, Satoru Ishihara, <u>Yoichi Kurumida</u>, *Tomoshi Kameda. 
                            Computational design of Burkholderia cepacia lipase mutants that show enhanced stereoselectivity in the production of L-menthol. <b><i>Journal of Agricultural and Food Chemistry</i></b>, 73(8):4829–4839, 2025. doi: <a target="_blank" href="https://doi.org/10.1021/acs.jafc.4c09949">10.1021/acs.jafc.4c09949</a>.
                        </li>
                        <li>
                            Jumpei Maki, Asami Oshimura, Yudai Shiotani, Maki Yamanaka, Sogen Okuda, Ryo C. Yanagita, Shigeru Kitani, Yasuhiro Igarashi, <u>Yutaka Saito</u>, Yasubumi Sakakibara, *Chihiro Tsukano, Kazuhiro Irie.
                            Validation of machine learning-assisted screening of PKC ligands: PKC binding affinity and activation. <b><i>Bioscience, Biotechnology, and Biochemistry</i></b>, zbaf008, 2025. doi: <a target="_blank" href="https://doi.org/10.1093/bbb/zbaf008">10.1093/bbb/zbaf008</a>.
                        </li>
                    </ul>
                    <h3>2024</h3>
                    <ul>
                        <li>
                            <u style="text-underline:double">Hongyi Shen</u>, <u>*Yutaka Saito</u>.
                            Protein-compound interaction prediction using microbial chemical communication network. <b><i>IPSJ Transactions on Bioinformatics</i></b>, 17(1):27-32, 2024. doi: <a target="_blank" href="https://doi.org/10.2197/ipsjtbio.17.27">10.2197/ipsjtbio.17.27</a>.
                        </li>
                        <li>
                            Takuma Matsushita, Shinji Kishimoto, Kodai Hara, Hiroshi Hashimoto, Hideki Yamaguchi, <u>Yutaka Saito</u>, *Kenji Watanabe.
                            Functional enhancement of flavin-containing monooxygenase through machine learning methodology. <b><i>ACS Catalysis</i></b>, 14(9):6945-6951, 2024. doi: <a target="_blank" href="https://doi.org/10.1021/acscatal.4c00826">10.1021/acscatal.4c00826</a>.
                        </li>
                        <li>
                            Andrejs Tučs, Tomoyuki Ito, <u style="text-underline:double">Yoichi Kurumida</u>, Sakiya Kawada, Hikaru Nakazawa, <u>Yutaka Saito</u>, *Mitsuo Umetsu, *Koji Tsuda.
                            Extensive antibody search with whole spectrum black-box optimization. <b><i>Scientific Reports</i></b>, 14(1):552, 2024. doi: <a target="_blank" href="https://doi.org/10.1038/s41598-023-51095-z">10.1038/s41598-023-51095-z</a>.
                        </li>
                        <li>
                            山口 秀輝, <u>齋藤 裕</u>. AlphaMissenseによる変異導入効果予測. 実験医学別冊 AlphaFold時代の構造バイオインフォマティクス, 羊土社, 2024. ISBN: 9784758122764.
                        </li>
                    </ul>
                    <h3>2023</h3>
                    <ul>
                        <li>
                            <u style="text-underline:double">Bian Bian</u>, Toshitaka Kumagai, <u>*Yutaka Saito</u>.
                            VeloPro: A pipeline integrating Ribo-seq and AlphaFold deciphers association patterns between translation velocity and protein structure features. <b><i>iMeta</i></b>, 2(4):e148, 2023. doi: <a target="_blank" href="https://doi.org/10.1002/imt2.148">10.1002/imt2.148</a>.
                        </li>
                        <li>
                            Yutaro Kumagai, <u>Yutaka Saito</u>, Yasuyuki S. Kida.
                            A multiomics atlas of brown adipose tissue development over time. <b><i>Endocrinology</i></b>, 164(6):bqad064, 2023. doi: <a target="_blank" href="https://doi.org/10.1210/endocr/bqad064">10.1210/endocr/bqad064</a>.
                        </li>
                        <li>
                            Yuki Ogawa, <u>Yutaka Saito (Joint First Author)</u>, <u style="text-underline:double">Hideki Yamaguchi</u>, Yohei Katsuyama, Yasuo Ohnishi.
                            Engineering the substrate specificity of toluene degrading enzyme XylM using biosensor XylS and machine learning. <b><i>ACS Synthetic Biology</i></b>, 12(2):572-582, 2023. doi: <a target="_blank" href="https://doi.org/10.1021/acssynbio.2c00577">10.1021/acssynbio.2c00577</a>.
                        </li>
                        <li>
                            Tomoyuki Ito, Thuy Duong Nguyen, <u>Yutaka Saito</u>, Yoichi Kurumida, Hikaru Nakazawa, Sakiya Kawada, Hafumi Nishi, Koji Tsuda, *Tomoshi Kameda, *Mitsuo Umetsu. 
                            Selection of target-binding proteins from the information of weakly enriched phage display libraries by deep sequencing and machine learning. <b><i>mAbs</i></b>, 15(1):2168470, 2023. doi: <a target="_blank" href="https://doi.org/10.1080/19420862.2023.2168470">10.1080/19420862.2023.2168470</a>.
                        </li>
                        <li>
                            <u>*山口 秀輝</u>, <u>齋藤 裕</u>. タンパク質の言語モデル. JSBi Bioinformatics Review, 4(1):52-67, 2023. <a target="_blank" href="https://doi.org/10.11234/jsbibr.2023.1">doi: 10.11234/jsbibr.2023.1</a>.
                        </li>
                        <li>
                            *梅津 光央, <u>齋藤 裕</u>, 亀田 倫史, 津田 宏治. 少ない実験データとベイズ最適化による機能タンパク質の配列設計. ケモインフォマティクスにおけるデータ収集の最適化と解析手法, 技術情報協会, 2023. ISBN: 9784861049446.
                        </li>
                    </ul>
                    <h3>2022</h3>
                    <ul>
                        <li>
                            <u style="text-underline:double">*Hideki Yamaguchi</u>, <u>Yutaka Saito</u>.
                            EvoOpt: an MSA-guided, fully unsupervised sequence optimization pipeline for protein design.
                            <b><i>Machine Learning in Structural Biology Workshop, The 36th Conference on Neural Information Processing Systems (NeurIPS 2022)</i></b>. [<a href="https://www.mlsb.io/">Conference Link</a>] [<a href="https://www.mlsb.io/papers_2022/EvoOpt_an_MSA_guided_fully_unsupervised_sequence_optimization_pipeline_for_protein_design.pdf">Paper PDF</a>]
                        </li>
                        <li>
                            Jumpei Maki, Asami Oshimura, Chihiro Tsukano, Ryo C Yanagita, <u>Yutaka Saito</u>, Yasubumi Sakakibara, *Kazuhiro Irie.
                            AI and computational chemistry-accelerated development of an alotaketal analogue with conventional PKC selectivity.
                            <b><i>Chemical Communications</i></b>, 58(47):6693-6696, 2022. doi: <a target="_blank" href="https://doi.org/10.1039/D2CC01759H">10.1039/D2CC01759H</a>. <a target="_blank" href="https://doi.org/10.26434/chemrxiv-2022-fnqnj">ChemRxiv</a>.
                        </li>
                        <li>
                            <u>*齋藤 裕</u>. 深層学習によるタンパク質の機能予測と設計. 生物工学会誌, 100(11):589-592, 2022. <a target="_blank" href="https://doi.org/10.34565/seibutsukogaku.100.11_589">doi: 10.34565/seibutsukogaku.100.11_589</a>.
                        </li>
                    </ul>
                    <h3>2021</h3>
                    <ul>
                        <li>
                            <u>Yutaka Saito</u>, Misaki Oikawa, Takumi Sato, Hikaru Nakazawa, Tomoyuki Ito, Tomoshi Kameda, *Koji Tsuda, *Mitsuo Umetsu.
                            Machine-learning-guided library design cycle for directed evolution of enzymes: the effects of training data composition on sequence space exploration.
                            <b><i>ACS Catalysis</i></b>, 11(23):14615–14624, 2021. doi: <a target="_blank" href="https://doi.org/10.1021/acscatal.1c03753">10.1021/acscatal.1c03753</a>.
                            <a target="_blank" href="https://doi.org/10.1101/2021.08.13.456323">bioRxiv</a>. <b>Selected as <a target="_blank" href="images/cover/accacs_v011i023-2.jpg">Cover Art</a></b>.
                        </li>
                        <li>
                            Godai Suzuki, <u>Yutaka Saito</u>, Motoaki Seki, Daniel Evans-Yamamoto, Mikiko Negishi, Kentaro Kakoi, Hiroki Kawai, Christian R Landry, *Nozomu Yachie, *Toutai Mitsuyama.
                            Machine learning approach for discrimination of genotypes based on bright-field cellular images.
                            <b><i>npj Systems Biology and Applications</i></b>, 7(1):31, 2021. doi: <a target="_blank" href="https://doi.org/10.1038/s41540-021-00190-w">10.1038/s41540-021-00190-w</a>.
                        </li>
                        <li>
                            <u style="text-underline:double">Hideki Yamaguchi</u>, <u>*Yutaka Saito</u>.
                            Evotuning protocols for Transformer-based variant effect prediction on multi-domain proteins.
                            <b><i>Briefings in Bioinformatics</i></b>, 22(6):bbab234, 2021. doi: <a target="_blank" href="https://doi.org/10.1093/bib/bbab234">10.1093/bib/bbab234</a>.
                            <a target="_blank" href="https://doi.org/10.1101/2021.03.05.434175">bioRxiv</a>.
                        </li>
                        <li>
                            Naoyuki Tajima, Toshitaka Kumagai, <u>Yutaka Saito</u>, *Tomoshi Kameda.
                            Comparative analysis of the relationship between translation efficiency and sequence features of endogenous proteins in multiple organisms.
                            <b><i>Genomics</i></b>, 113(4):2675-2682, 2021. doi: <a target="_blank" href="https://doi.org/10.1016/j.ygeno.2021.05.037">10.1016/j.ygeno.2021.05.037</a>.
                        </li>
                        <li>
                            Shin Irumagawa, Kaito Kobayashi, <u>Yutaka Saito</u>, Takeshi Miyata, Mitsuo Umetsu, Tomoshi Kameda, *Ryoichi Arai.
                            Rational thermostabilisation of four-helix bundle dimeric <i>de novo</i> proteins.
                            <b><i>Scientific Reports</i></b>, 11(1):7526, 2021. doi: <a target="_blank" href="https://doi.org/10.1038/s41598-021-86952-2">10.1038/s41598-021-86952-2</a>.
                        </li>
                        <li>
                            Yoshihiro Miyazaki, Tatsuya Oda, Yuki Inagaki, Hiroko Kushige, <u>Yutaka Saito</u>, Nobuhito Mori, Yuzo Takayama, Yutaro Kumagai, Toutai Mitsuyama, *Yasuyuki S. Kida.
                            Adipose-derived mesenchymal stem cells differentiate into heterogeneous cancer-associated fibroblasts in a stroma-rich xenograft model.
                            <b><i>Scientific Reports</i></b>, 11(1):4690, 2021. doi: <a target="_blank" href="https://doi.org/10.1038/s41598-021-84058-3">10.1038/s41598-021-84058-3</a>.
                        </li>
                        <li>
                            *梅津 光央, <u>齋藤 裕</u>, 亀田 倫史, 津田 宏治. 機械学習を用いたタンパク質デザイン. 生物物理, 61(3):177-179, 2021. <a target="_blank" href="https://doi.org/10.2142/biophys.61.177">doi: 10.2142/biophys.61.177</a>.
                        </li>
                        <li>
                            <u>*齋藤 裕</u>. ゲノムは遺伝子を単語とする文章である（か？） バイオサイエンスとインダストリー, 79(1):49, 2021.
                        </li>
                    </ul>
                    <h3>2020</h3>
                    <ul>
                        <li>
                            Yoichi Kurumida, <u>Yutaka Saito</u>, *Tomoshi Kameda.
                            Predicting antibody affinity changes upon mutations by combining multiple predictors.
                            <b><i>Scientific Reports</i></b>, 10(1):19533, 2020. doi: <a target="_blank" href="https://doi.org/10.1038/s41598-020-76369-8">10.1038/s41598-020-76369-8</a>.
                        </li>
                        <li>
                            Jong-Hun Lee, <u>Yutaka Saito</u>, Sung-Joon Park, *Kenta Nakai.
                            Existence and possible roles of independent non-CpG methylation in the mammalian brain.
                            <b><i>DNA Research</i></b>, 27(4):dsaa020, 2020. doi:
                            <a target="_blank" href="https://doi.org/10.1093/dnares/dsaa020">10.1093/dnares/dsaa020</a>.
                        </li>
                        <li>
                            <u>*齋藤 裕</u>. 機械学習による生体分子の機能改良. JSBi Bioinformatics Review, 1(1):12-17, 2020. <a target="_blank" href="https://doi.org/10.11234/jsbibr.2020.2">doi: 10.11234/jsbibr.2020.2</a>.
                        </li>
                        <li>
                            <u>*齋藤 裕</u>, 北川 航, 亀田 倫史. 放線菌におけるタンパク質生産量向上のための新規コドン最適化技術. バイオサイエンスとインダストリー, 78(5):412-413, 2020.
                        </li>
                    </ul>
                    <h3>2019</h3>
                    <ul>
                        <li>
                            Thuy Duong Nguyen, <u>Yutaka Saito</u>, *Tomoshi Kameda.
                            CodonAdjust: a software for <i>in silico</i> design of a mutagenesis library.
                            <b><i>Protein Engineering, Design, and Selection</i></b>, 32(11):503-511, 2019. doi: <a target="_blank" href="https://doi.org/10.1093/protein/gzaa013">10.1093/protein/gzaa013</a>.
                        </li>
                        <li>
                            *Ryuichiro Nakato, *Youichiro Wada, Ryo Nakaki, Genta Nagae, Yuki Katou, Shuichi Tsutsumi, Natsu Nakajima, Hiroshi Fukuhara, Atsushi Iguchi, Takahide Kohro, Yasuharu Kanki, <u>Yutaka Saito</u>, Mika Kobayashi, Akashi Izumi-Taguchi, Naoki Osato, Kenji Tatsuno, Asuka Kamio, Yoko Hayashi-Takanaka, Hiromi Wada, Shinzo Ohta, Masanori Aikawa, Hiroyuki Nakajima, Masaki Nakamura, Rebecca C. McGee, Kyle W. Heppner, Tatsuo Kawakatsu, Michiru Genno, Hiroshi Yanase, Haruki Kume, Takaaki Senbonmatsu, Yukio Homma, Shigeyuki Nishimura, Toutai Mitsuyama, Hiroyuki Aburatani, *Hiroshi Kimura, *Katsuhiko Shirahige.
                            Comprehensive epigenome characterization reveals diverse transcriptional regulation across human vascular endothelial cells.
                            <b><i>Epigenetics &amp; Chromatin</i></b>, 12:77, 2019. doi: <a target="_blank" href="https://doi.org/10.1186/s13072-019-0319-0">10.1186/s13072-019-0319-0</a>.
                        </li>
                        <li>
                            <u>Yutaka Saito</u>, Wataru Kitagawa, Toshitaka Kumagai, Naoyuki Tajima, Yoshiyuki Nishimiya, Koichi Tamano, Yoshiaki Yasutake, *Tomohiro Tamura, *Tomoshi Kameda.
                            Developing a codon optimization method for improved expression of recombinant proteins in actinobacteria.
                            <b><i>Scientific Reports</i></b>, 9(1):8338,
                            2019. doi: <a target="_blank" href="https://doi.org/10.1038/s41598-019-44500-z">10.1038/s41598-019-44500-z</a>.
                        </li>
                        <li>
                            *梅津 光央, <u>齋藤 裕</u>, 亀田 倫史, 津田 宏治. 機械学習が道先案内するタンパク質の進化分子工学. BIO INDUSTRY, 36(12):55-63, 2019.
                        </li>
                        <li>
                            亀田 倫史, <u>齋藤 裕</u>, 及川 未早来, *梅津 光央, 津田 宏治. 機械学習支援による蛋白質高機能化. アンサンブル：分子シミュレーション研究会誌, 21(1):34-38, 2019.
                        </li>
                    </ul>
                    <h3>2018</h3>
                    <ul>
                        <li>
                            Maya Hirohara, <u>Yutaka Saito</u>, Yuki Koda, Kengo Sato, *Yasubumi Sakakibara.
                            Convolutional neural network based on SMILES representation of compounds for detecting chemical motif.
                            <b><i>BMC Bioinformatics</i></b>, 19(Suppl 19):526, 2018. doi: <a target="_blank" href="https://doi.org/10.1186/s12859-018-2523-5">10.1186/s12859-018-2523-5</a>.
                        </li>
                        <li>
                            <u>Yutaka Saito</u>, Misaki Oikawa, Hikaru Nakazawa, Teppei Niide, Tomoshi Kameda, *Koji Tsuda, *Mitsuo Umetsu.
                            Machine-learning-guided mutagenesis for directed evolution of fluorescent proteins. <b><i>ACS Synthetic Biology</i></b>, 7(9):2014-2022,
                            2018. doi: <a target="_blank" href="https://doi.org/10.1021/acssynbio.8b00155">10.1021/acssynbio.8b00155</a>. <a target="_blank" href="https://doi.org/10.1101/282996">bioRxiv</a>.
                        </li>
                        <li>
                            Naofumi Ito, Kaoru Katoh, Hiroko Kushige, <u>Yutaka Saito</u>, Terumasa Umemoto, Yu Matsuzaki, Hiroshi Kiyonari, Daiki Kobayashi, Minami Soga, Takumi Era, Norie Araki, Yasuhide Furuta, Toshio Suda, Yasuyuki Kida, *Kunimasa Ohta.
                            Ribosome incorporation into somatic cells promotes transdifferentiation towards multipotency.
                            <b><i>Scientific Reports</i></b>, 8(1):1634, 2018. doi: <a target="_blank" href="https://doi.org/10.1038/s41598-018-20057-1">10.1038/s41598-018-20057-1</a>.
                        </li>
                        <li>
                            *亀田 倫史, <u>齋藤 裕</u>, 田島 直幸, 西宮 佳志, 玉野 孝一, 北川 航, 安武 義晃, 田村 具博. 情報解析に基づく遺伝子配列改変による発現量調節. スマートセルインダストリー -微生物細胞を用いた物質生産の展望-, シーエムシー出版, 2018. ISBN: 9784781313344.
                        </li>
                        <li>
                            <u>*Yutaka Saito</u>. Comparative Epigenomics. Encyclopedia of Bioinformatics and Computational Biology, Elsevier, 2018. ISBN: 9780128114322.
                        </li>
                    </ul>
                    <h3>2017</h3>
                    <ul>                    
                        <li>
                            <u>Yutaka Saito</u>, Chie Sugimoto, Toutai Mituyama, *Hiroshi Wakao.
                            Epigenetic silencing of V(D)J recombination is a major determinant for selective differentiation of mucosal-associated invariant T cells from induced pluripotent stem cells.
                            <b><i>PLOS ONE</i></b>, 12(3):e0174699, 2017. doi:<a target="_blank" href="https://doi.org/10.1371/journal.pone.0174699">10.1371/journal.pone.0174699</a>.
                        </li>
                        <li>
                            Yuzo Takayama, Tamami Wakabayashi, Hiroko Kushige, <u>Yutaka Saito</u>, Yoichiro Shibuya, Shinsuke Shibata, Wado Akamatsu, Hideyuki Okano, *Yasuyuki S. Kida.
                            Brief exposure to small molecules allows induction of mouse embryonic fibroblasts into neural crest-like precursors.
                            <b><i>FEBS Letters</i></b>, 591(4):590-602, 2017. doi: <a target="_blank" href="https://doi.org/10.1002/1873-3468.12572">10.1002/1873-3468.12572</a>.
                        </li>
                    </ul>
                    <h3>Before 2016</h3>
                    <ul>
                        <li>
                            <u>Yutaka Saito</u>, *Toutai Mituyama.
                            Detection of differentially methylated regions from bisulfite-seq data by hidden Markov models incorporating genome-wide methylation level distributions.
                            <b><i>BMC Genomics</i></b>, 16(Suppl 12):S3, 2015. doi: <a target="_blank" href="https://doi.org/10.1186/1471-2164-16-S12-S3">10.1186/1471-2164-16-S12-S3</a>.
                        </li>
                        <li>
                            Hitomi Takada, <u>Yutaka Saito (Joint First Author)</u>, Toutai Mituyama, Zong Wei, Eiji Yoshihara, Sandra Jacinto, Michael Downes, Ronald M. Evans, *Yasuyuki S. Kida.
                            Methylome, transcriptome, and PPARγ cistrome analyses reveal two epigenetic transitions in fat cells.
                            <b><i>Epigenetics</i></b>,9(9):1195-1206, 2014. doi: <a target="_blank" href="https://doi.org/10.4161/epi.29856">10.4161/epi.29856</a>.
                        </li>
                        <li>
                            <u>Yutaka Saito</u>, Junko Tsuji, *Toutai Mituyama.
                            Bisulfighter: accurate detection of methylated cytosines and differentially methylated regions. <b><i>Nucleic Acids Research</i></b>, 42(6):e45, 2014. doi: <a target="_blank" href="https://doi.org/10.1093/nar/gkt1373">10.1093/nar/gkt1373</a>
                        </li>
                        <li>
                            Masaomi Nakamura, Tsuyoshi Hachiya, <u>Yutaka Saito</u>, Kengo Sato, *Yasubumi Sakakibara.
                            An efficient algorithm for de novo predictions of biochemical pathways between chemical compounds.
                            <b><i>BMC Bioinformatics</i></b>, 13(Suppl 17):S8, 2012. doi: <a target="_blank" href="https://doi.org/10.1186/1471-2105-13-S17-S8">10.1186/1471-2105-13-S17-S8</a>.
                        </li>
                        <li>
                            Yohei Okada, <u>Yutaka Saito</u>, Kengo
                            Sato, *Yasubumi Sakakibara. Improved measurements of RNA structure conservation
                            with generalized centroid estimators. <b><i>Frontiers in Genetics</i></b>, 2:54, 2011. doi:
                            <a target="_blank" href="https://doi.org/10.3389/fgene.2011.00054">10.3389/fgene.2011.00054</a>.
                        </li>
                        <li>
                            <u>Yutaka Saito</u>, Kengo Sato, *Yasubumi Sakakibara.
                            Fast and accurate clustering of noncoding RNAs using ensembles of sequence alignments and secondary structures.
                            <b><i>BMC Bioinformatics</i></b>, 12(Suppl 1):S48, 2011. doi: <a target="_blank" href="https://doi.org/10.1186/1471-2105-12-S1-S48">10.1186/1471-2105-12-S1-S48</a>.
                        </li>
                        <li>
                            <u>Yutaka Saito</u>, Kengo Sato, *Yasubumi Sakakibara.
                            Robust and accurate prediction of noncoding RNAs from aligned sequences.
                            <b><i>BMC Bioinformatics</i></b>, 11(Suppl 7):S3, 2010. doi: <a target="_blank" href="https://doi.org/10.1186/1471-2105-11-S7-S3">10.1186/1471-2105-11-S7-S3</a>.
                        </li>
                        <li>
                            Kengo Sato, <u>Yutaka Saito</u>, *Yasubumi Sakakibara.
                            Gradient-based optimization of hyperparameters for base-pairing profile local alignment kernels.
                            <b><i>Genome Informatics</i></b>, 23(1):128-38, 2009. doi: <a target="_blank" href="https://doi.org/10.1142/9781848165632_0012">10.1142/9781848165632_0012</a>.
                        </li>
                        <li>
                            Kensuke Morita, <u>Yutaka Saito (Joint First Author)</u>, Kengo Sato, Kotaro Oka, Kohji Hotta, *Yasubumi Sakakibara.
                            Genome-wide searching with base-pairing kernel functions for noncoding RNAs: computational and expression analysis of snoRNA families in <i>Caenorhabditis elegans</i>.
                            <b><i>Nucleic Acids Research</i></b>, 37(3):999-1009, 2009. doi: <a target="_blank" href="https://doi.org/10.1093/nar/gkn1054">10.1093/nar/gkn1054</a>.
                        </li>
                        <li>
                            Kengo Sato, <u>Yutaka Saito</u>, *Yasubumi Sakakibara. Base-pairing profile local alignment kernels for functional RNA analyses. 情報処理学会研究報告バイオ情報学（BIO）, 2009-BIO-17(8):1-7, 2009.
                        </li>
                    </ul>

                    <h2>Media coverages</h2>
                    <ul>
                        <li>
                            生成AIでたんぱく質設計 開発競争が本格化「大戦国時代」, 科学の森, <a target="_blank" href="https://mainichi.jp/articles/20240321/ddm/016/040/019000c">毎日新聞</a> (朝刊15面, 3/21, 2024);<br>
                        </li>
                        <li>
                            <b><i>ACS Synthetic Biology</i></b>, 12(2):572-582, 2023. doi: <a target="_blank" href="https://doi.org/10.1021/acssynbio.2c00577">10.1021/acssynbio.2c00577</a>.<br>
                            雑誌: タンパク質を語る言語, 大規模言語モデル 科学を変えるAI, <a target="_blank" href="https://www.nikkei-science.com/page/magazine/202310.html">日経サイエンス</a>;<br>
                        </li>
                        <li>
                            <b><i>ACS Catalysis</i></b>, 11(23):14615–14624, 2021. doi: <a target="_blank" href="https://doi.org/10.1021/acscatal.1c03753">10.1021/acscatal.1c03753</a>.<br>
                            Web news: <a target="_blank" href="https://www.nikkei.com/article/DGXLRSP622897_S1A201C2000000/">日本経済新聞電子版</a>;<br>
                            プレスリリース: <a target="_blank" href="https://www.tohoku.ac.jp/japanese/newimg/pressimg/tohokuuniv-press20211202_01web_ai.pdf">東北大</a>; <a target="_blank" href="https://www.riken.jp/press/2021/20211202_4/index.html">理研</a>;<br>
                        </li>
                        <li>
                            <b><i>Scientific Reports</i></b>, 9(1):8338, 2019. <a target="_blank" href="https://doi.org/10.1038/s41598-019-44500-z">doi: 10.1038/s41598-019-44500-z</a>.<br>
                            プレスリリース: <a target="_blank" href="https://www.aist.go.jp/aist_j/press_release/pr2019/pr20190606/pr20190606.html">産総研</a>;<br>
                            新聞: 化学工業日報 (朝刊3面, 6/14, 2019); 科学新聞 (朝刊1面, 6/28, 2019); 薬事日報 (朝刊17面, 7/17, 2019);
                        </li>
                        <li>
                            <b><i>ACS Synthetic Biology</i></b>, 7(9):2014-2022, 2018. <a target="_blank" href="https://doi.org/10.1021/acssynbio.8b00155">doi: 10.1021/acssynbio.8b00155</a>.<br>
                            Research highlight: <a target="_blank" href="https://twitter.com/NatureBiotech/status/1034078872759885824">Nature Biotechnology</a>;<br>
                            Web news: <a target="_blank" href="https://www.nikkei.com/article/DGXLRSP489285_R30C18A8000000/">日本経済新聞電子版</a>; <a target="_blank" href="https://bio.nikkeibp.co.jp/atcl/release/18/09/03/06045/">日経バイオテクONLINE</a>; <a target="_blank" href="https://news.mynavi.jp/article/20180831-686882/">マイナビニュース</a>; etc.<br>
                            プレスリリース: <a target="_blank" href="https://www.tohoku.ac.jp/japanese/newimg/pressimg/tohokuuniv-press20180831_01web_tanpaku.pdf">東北大学</a>; <a target="_blank" href="https://www.aist.go.jp/aist_j/press_release/pr2018/pr20180831/pr20180831.html">産総研</a>; <a target="_blank" href="https://www.riken.jp/press/2018/20180831_1/">理研</a>; <a target="_blank" href="https://www.sbj.or.jp/2018/news/news_20180906-1.html">日本生物工学会</a>;<br>
                            新聞: 化学工業日報 (朝刊6面, 9/3, 2018); <a target="_blank" href="https://www.nikkan.co.jp/articles/view/00487449">日刊工業新聞</a> (朝刊23面, 9/6, 2018);
                        </li>
                        <li>
                            <b><i>Scientific Reports</i></b>, 8(1):1634, 2018. <a target="_blank" href="https://doi.org/10.1038/s41598-018-20057-1">doi: 10.1038/s41598-018-20057-1</a>.<br>
                            Research highlight: <a target="_blank" href="https://facultyopinions.com/prime/732574095">F1000Prime</a>;<br>
                            Web news: <a target="_blank" href="https://www.nikkei.com/article/DGXMZO27374530V20C18A2TJM000/">日本経済新聞電子版</a>; <a target="_blank" href="https://www.eurekalert.org/news-releases/538995">EurekAlert!</a>; <a target="_blank" href="https://phys.org/news/2018-02-ribosomes-somatic-cell-pluripotency.html">Phys.org</a>; etc.<br>
                            プレスリリース: <a target="_blank" href="https://www.kumamoto-u.ac.jp/daigakujouhou/kouhou/pressrelease/2017-file/release180126.pdf">熊本大学</a>;<br>
                            新聞: 熊本日日新聞 (1/27, 2018);
                        </li>
                        <li>
                            <b><i>Nucleic Acids Research</i></b>, 37(3):999-1009, 2009. <a target="_blank" href="https://doi.org/0.1093/nar/gkn1054">doi: 10.1093/nar/gkn1054</a>.<br>
                            Web news: <a target="_blank" href="https://www.genomeweb.com/gene-silencinggene-editing/quest-function">genomeweb</a>;
                        </li>
                    </ul>

                </article>

            </div>

            <footer>

                <div id="footerarea">

                    <p><span class="lar">北里大学・産業技術総合研究所・東京大学</span></p>
                    <p><span class="huge">齋藤研究室</span></p>

                    <p><span class="mid2">Saito Lab</span></p>

                </div>

                <div id="copy">Copyright &copy; Saito Lab</div>

            </footer>

            <a href="#toparea" id="gototop" title="トップへのリンク"><img src="./images/top.png" alt="トップへ戻るボタン"></a>

        </div>

        <div id="overlay" class="hidden-md hidden-lg"></div>

    </div>

    <script src="./slick/slick.min.js"></script>


    <script>
        jQuery(function() {
            jQuery('.visualarea').slick({
                infinite: true,
                dots: true,
                slidesToShow: 1,
                slidesToScroll: 1,
                autoplay: true,
                fade: true,
                easing: 'linear',
                speed: 3000,
                autoplaySpeed: 5000,
                arrows: false,

                responsive: [{
                        breakpoint: 1200,
                        settings: {
                            slidesToShow: 1,
                            slidesToScroll: 1,
                        }
                    },
                    {
                        breakpoint: 600,
                        settings: {
                            slidesToShow: 1,
                            slidesToScroll: 1,
                        }
                    }
                ]
            });
        });

    </script>

    <script>
        jQuery(function() {
            var gotoTop = jQuery('#gototop');

            gotoTop.hide();


            jQuery(window).scroll(function() {
                if (jQuery(this).scrollTop() > 300) {
                    gotoTop.fadeIn();
                } else {
                    gotoTop.fadeOut();
                }
            });

            gotoTop.click(function() {
                jQuery('body, html').animate({
                    scrollTop: 0
                }, 200, 'swing');
                return false;
            });


        });

    </script>


    <script>
        var ua = navigator.userAgent.toLowerCase();
        var isMobile = /iphone/.test(ua) || /android(.+)?mobile/.test(ua);

        if (!isMobile) {
            jQuery('a[href^="tel:"]').on('click', function(e) {
                e.preventDefault();
            });
        }

    </script>

    <script>
        jQuery(document).ready(function() {
            jQuery('#menubtn').on('click', function() {
                jQuery('#mainnavigation').slideToggle();
            });
        });

    </script>

    <script>
        jQuery(function() {
            var touch = false;
            jQuery('#humberger').on('click touchstart', function(e) {
                switch (e.type) {
                    case 'touchstart':
                        drawerToggle();
                        touch = true;
                        return false;
                        break;
                    case 'click':
                        if (!touch)
                            drawerToggle();
                        return false;
                        break;
                }

                function drawerToggle() {
                    jQuery('body').toggleClass('drawer-opened');
                    touch = false;
                }
            })
            jQuery('#overlay').on('click touchstart', function() {
                jQuery('body').removeClass('drawer-opened');
            })

        });

    </script>

</body>

</html>
